java/build-android.gradle

```diff
@@ -26,14 +29,22 @@ project.version = rootProject.file('../VERSION_NUMBER').text.trim()
 project.group = "com.microsoft.onnxruntime"
 
 def tmpArtifactId = enableTrainingApis ? project.name + "-training" : project.name
-def mavenArtifactId = tmpArtifactId + '-android'
+def mavenArtifactId = tmpArtifactId + '-android' + (qnnVersion != null ? '-qnn' : '')
+
+//should the mavenArtifactId be read from the packageName variable as
+//that's how it's used in the build_aar_copy_artifacts.sh while copying the artifacts
```

Comment:
I think we generally don't want to assume that there's a PACKAGE_NAME environment variable outside the CI build environment, e.g., when doing a local build. that said, we can probably do better than defining it in multiple places. optional to fix it in this PR though :)